gym/envs/box2d/bipedal_walker.py

```diff
@@ -586,8 +591,11 @@ def step(self, action: np.ndarray):
         return np.array(state, dtype=np.float32), reward, done, {}
 
     def render(self, mode: str = "human"):
-        import pygame
-        from pygame import gfxdraw
+        try:
+            import pygame
+            from pygame import gfxdraw
+        except ImportError:
+            raise ImportError("Pygame is not installed, run `pip install gym[box2d]`")
```

Comment:
Capitalization of first letter of error message inconsistent with other error messages.

Reply:
Thanks for the spot, I will revert to the rest of the code base

Reply:
I just meant within the PR, since some error messages within the PR were capitalized and some weren't. I'm not sure what the standard is across gym.

Reply:
Ohh, I realised that when implementing the fix
Personally, I dont mind, just agree that it should be consistent
I changed it to lower case on a coin flip. Can change it to upper case if anyone has any major issues with that